### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by inyongnation** [ztjhz/BetterChatGPT#494] [General]
{
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details.",
        "type": "insufficient_quota",
        "param": null,
        "code": "insufficient_quota"
    }
}

Message from Better ChatGPT:
We recommend changing your API endpoint or API key

**Comment by zzz6519003** [comment]
me too

**Reply by ztjhz** [reply]
Your OpenAI account ran out of money for the API key.

**Comment by ztjhz** [comment]
Your OpenAI account ran out of money for the API key.
